.github/workflows/dev-image-update.yaml

```diff
@@ -165,7 +165,7 @@ jobs:
 
   action_notify:
     runs-on: ubuntu-latest
-    if: always()
+    if: ${{ always() && !contains( github.event.sender.login, 'broadbot') }}
```

Comment:
Is there a way to add a comment around why we have that check?

Reply:
Good point - I added the comment to the first mentioned of the logic at the top of the file.